app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,97 +1,143 @@
 import React, { useState, useEffect } from "react";
 
+import { Search, Settings, Plus, BurgerMenu } from "@bigbinary/neeto-icons";
+import { Typography, Input, Button } from "@bigbinary/neetoui/v2";
+import { MenuBar, Header } from "@bigbinary/neetoui/v2/layouts";
```

Comment:
Please use alias for `@bigbinary/neetoui/v2` as `neetoui`.

Reply:
Done.